include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Ok, I can't figure out (yet) how this could work without #defining USE_LONG_LONG (or, in other words, by moving the usage to the cpp file). I won't ask you to put the earlier things back, let me later (tonight, my time) clean up this mess I created :)